core/src/main/scala/slamdata/SbtSlamDataBase.scala

```diff
@@ -507,62 +507,69 @@ abstract class SbtSlamDataBase extends AutoPlugin {
             else if (isBreaking) "breaking"
             else "feature"
 
-          val dependencyCheck = repo.updates
-            .map(u => s"${u.dependency.organization}:${u.dependency.name}:${u.newRevision}")
-            .mkString("trickleCheckVersion ", " ", "")
-          
-          if (runWithLoggerSeq(Seq("sbt", dependencyCheck), log, merge=true, Some(dirFile)) != 0) {
-            sys.error(s"repository ${repo.repository} did not apply versions file correctly")
-          }
+          if (runWithLoggerSeq(Seq("sbt", "update"), log, merge=true, Some(dirFile)) != 0) {
```

Comment:
I'd do trickle check then update, since trickle check is based entirely on settings, while update goes to the network.